File: snapshots/output-net10.0/syntax/Main/Packages.cs
  using System.Diagnostics.CodeAnalysis;
//      ^^^^^^ reference scip-dotnet nuget . . System/
//             ^^^^^^^^^^^ reference scip-dotnet nuget . . Diagnostics/
//                         ^^^^^^^^^^^^ reference scip-dotnet nuget . . CodeAnalysis/
  using DiffPlex.DiffBuilder;
  using DiffPlex.DiffBuilder.Model;

  namespace Main;
//          ^^^^ reference scip-dotnet nuget . . Main/

  [SuppressMessage("ReSharper", "all")]
// ^^^^^^^^^^^^^^^ reference scip-dotnet nuget System.Runtime 10.0.0.0 CodeAnalysis/SuppressMessageAttribute#`.ctor`().
  public class Packages
//             ^^^^^^^^ definition scip-dotnet nuget . . Main/Packages#
//                      documentation ```cs\nclass Packages\n```
//                      enclosing_range 6:0 13:1
  {
      DiffPaneModel Diff()
//                  ^^^^ definition scip-dotnet nuget . . Main/Packages#Diff().
//                       documentation ```cs\nprivate DiffPaneModel Packages.Diff()\n```
//                       enclosing_range 9:4 12:5
      {
          return InlineDiffBuilder.Diff("a", "b");
      }
  }

File: snapshots/output-net10.0/syntax/VBMain/Packages.vb
  Imports System.Diagnostics.CodeAnalysis
'         ^^^^^^ reference scip-dotnet nuget . . System/
'                ^^^^^^^^^^^ reference scip-dotnet nuget . . Diagnostics/
'                            ^^^^^^^^^^^^ reference scip-dotnet nuget . . CodeAnalysis/
  Imports DiffPlex.DiffBuilder
  Imports DiffPlex.DiffBuilder.Model

  Namespace VBMain
'           ^^^^^^ reference scip-dotnet nuget . . VBMain/
      <SuppressMessage("ReSharper", "all")>
'      ^^^^^^^^^^^^^^^ reference scip-dotnet nuget System.Runtime 10.0.0.0 CodeAnalysis/SuppressMessageAttribute#`.ctor`().
      Public Class Packages
'                  ^^^^^^^^ definition scip-dotnet nuget . . VBMain/Packages#
'                           documentation ```vb\nClass Packages\n```
'                           enclosing_range 5:4 10:13
          Private Function Diff() As DiffPaneModel
'                          ^^^^ definition scip-dotnet nuget . . VBMain/Packages#Diff().
'                               documentation ```vb\nPrivate Function Packages.Diff() As DiffPaneModel\n```
'                               enclosing_range 7:8 9:20
              Return InlineDiffBuilder.Diff("a", "b")
          End Function
      End Class
  End Namespace
